test/transactions/4_registerMultisignatureAccount.js

```diff
@@ -28,6 +29,12 @@ describe('#registerMultisignatureAccount transaction', () => {
 	const timeWithOffset = 38350076;
 	const lifetime = 5;
 	const minimum = 2;
+	const tooShortPublicKeyKeysgroup = [
+		'd019a4b6fa37e8ebeb64766c7b239d962fb3b3f265b8d3083206097b912cd9',
+	];
+	const plusPrependedPublicKeyKeysgroup = [
```

Comment:
Since these are arrays it would also make sense to assign them in the `beforeEach` hook like `keysgroup`